department-of-veterans-affairs/caseflow
# Justin/appeals 43606

Resolves [Jira Issue Title](https://jira.devops.va.gov/browse/JIRA-12345)

# Description
Please explain the changes you made here.

## Acceptance Criteria
- [ ] Inline styles are removed from Correspondence related frontend code and moved into styles/queue/_correspondence.scss.

Files included in client/app/queue/correspondence folder for Review Package
- [x] PackageActionModal.jsx
- [x] CorrespondencePdfFooter.jsx
- [x] CorrespondencePdfToolbar.jsx
- [x] CmpDocuments.jsx
- [x] CorrespondenceReviewPackage.jsx
- [x] ReviewForm.jsx
- [x] ReviewPackageCaseTitle.jsx
- [x] ReviewPackageData.jsx

## Testing Plan

Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below

1. Log in as Jolly postman and go to your queue. Click the switch views dropdown and go to "your correspondence".
2. Open a batman correspondence (such as michael keaton).
3. The most important part is to verify that the screens appear as expected from the wireframes. But the CSS can also be tested at a more granular level to verify the work has been completed correctly. To do this, read on!
4. Open Dev tools (use ctrl+shift+i on windows)
5. Open the Elements pane, and then click the "select elements" button (alternatively, this can easily be done in one step with ctrl+shift+c on windows)
6. Click on the text "Review Package" and verify that there is a "class" name beginning with correspondence after the "div" and the "h1" (see right pane in photo below). This verifies ReviewPackageCaseTitle has been worked.
![image](https://github.com/department-of-veterans-affairs/caseflow/assets/95875751/ee590676-726c-418a-8060-e2c348480569)
7. Next, click on the text "CMP Information" and verify that there is a "class" name beginning with correspondence after the "div" and the "h2" tags. This verifies ReviewPackageData has been worked.
8. Next, click in the empty space just below "General Information", but above the gray box. Verify the "div" element has a class beginning with "review-form". This verifies "ReviewForm" has been worked.
